openedx/core/djangoapps/theming/core.py

```diff
@@ -21,9 +22,14 @@ def enable_theming():
         )
 
     for theme in get_themes():
-        locale_dir = theme.path / "conf" / "locale"
-        if locale_dir.isdir():
-            settings.LOCALE_PATHS = (locale_dir, ) + settings.LOCALE_PATHS
-
         if theme.themes_base_dir not in settings.MAKO_TEMPLATES['main']:
             settings.MAKO_TEMPLATES['main'].insert(0, theme.themes_base_dir)
+
+
+def add_theming_locales():
```

Comment:
Should this be a private function called by `enable_theming()` or would we ever want to call `enable_theming()` and `add_theming_locales()` separately?